source/documentation/using_ci/index.md

```diff
@@ -0,0 +1,13 @@
+# Configuring a custom continuous integration (CI) system
+
+Setting up a CI service lets you make sure your code changes are delivering new features without introducing new bugs or breaking existing functionality. This helps you to iterate code faster and with greater confidence, a huge benefit once the codebase becomes larger and more complex. 
+ 
+For advanced projects that need high levels of control over configuration, a tool like [Jenkins](https://jenkins.io/) is often used. For simpler projects, or where a team needs to manage deployments of prototypes from their github repositories rather than local code, cloud-based services such as [Travis CI](https://travis-ci.com/) or [CircleCI](https://circleci.com/) are common. Instructions to configure both Jenkins and Travis to run on GOV.UK PaaS are below.
```

Comment:
I don't think we are recommending it, just saying this is what people do. There are two comon use-cases: Proper TAs wanting to do Proper CI (who might or might not use Jenkins, and will Have Opinions), and designers/tiny teams who just want a quick way to link up deployment to github. As the latter came up in user research, I want something to be able to link out to, in order to reassure those more advanced prototypers.